resources/build/history/validate-history.sh

```diff
@@ -1,3 +1,6 @@
+echo "**TODO** Update script to work with new history/versioning"
+exit 1
+
```

Comment:
Viewing from Linux, the following scripts lost their executable flag
/resources/build/history/*.sh
/linux/scripts/version.sh